spec/factories/users.rb

```diff
@@ -19,6 +19,11 @@
       factory :developer do
         developer { true }
       end
+      factory :ambassador do
```

Comment:
... this isn't the naming convention that exists for everyting, but I decided a little while ago that I'd prefer if factories were prefixed with the name of the parent factory - so this would be `user_ambassador`. At some point, it would be great if we updated the existing ones, but that's not a big priority.

It just helps keep things on track. This goes for the other factories as well